addons/account/models/chart_template.py

```diff
@@ -333,10 +333,12 @@ def tax_template_changed(tax, template):
                                       and t.type_tax_use == values.get('type_tax_use')\
                                       and t.tax_scope == values.get('tax_scope', False)
                             )
-                        uniq_key = unique_tax_name_key(oldtax)
-                        rename_idx = len(list(filter(lambda t: re.match(fr"^(?:\[old\d*\] |){uniq_key[0]}$", t[0]) and t[1:] == uniq_key[1:], unique_tax_name_keys)))
-                        if rename_idx:
-                            oldtax.name = f"[old{rename_idx - 1 if rename_idx > 1 else ''}] {oldtax.name}"
+                        uniq_key = unique_tax_name_key(oldtax[0] if len(oldtax) > 1 else oldtax)
```

Comment:
```suggestion
                        uniq_key = unique_tax_name_key(oldtax[0])
```

Reply:
To protect it in case oldtax is empty. If the record by that name is missing it will be empty.